### PR TITLE
(DOCSP-12824) Manually Deleting an Object With A Relationship [Node/React Native]

### PR DESCRIPTION
**Pull request by mongomoe**
### Issue JIRA link:
https://jira.mongodb.org/browse/DOCSP-12824

### Docs staging link (requires sign-in on MongoDB Corp SSO):
Node: https://docs-mongodbcom-staging.corp.mongodb.com/realm/mohammad.hunan/DOCSP-12824-Cascading-Deletes/node/writes.html#chaining-delete
React Native: https://docs-mongodbcom-staging.corp.mongodb.com/realm/mohammad.hunan/DOCSP-12824-Cascading-Deletes/react-native/writes.html#manually-delete-an-object-with-a-relationship

Note, I created a follow-on ticket to update the entire writes page to be a little less confusing by having the object models defined on the page. I didn't want to scope creep. This is that ticket: https://jira.mongodb.org/browse/DOCSP-13145

### CODE REVIEW

File: source/node/writes.txt [hunk 1]
```diff
@@ -245,30 +245,32 @@ collection itself.
...
 
-Cascading Delete
-~~~~~~~~~~~~~~~~
+Manually Delete An Object with a Relationship
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+When you delete an object with a :ref:`relationship <node-client-relationships>`
+to another object, Realm does not automatically delete the other object. To
+delete an object with a relationship to another object, you must manually delete
+the object.
```
Comment (terakilobyte) [inline source/node/writes.txt:256]: The wording of this is very confusing, imo. I had to check the JIRA ticket to fully understand the issue being resolved.
```suggestion
When you delete a non-embedded object with a :ref:`relationship <node-client-relationships>`
to a child object, Realm does not automatically delete the child object.
```

**Review by terakilobyte — CHANGES_REQUESTED** [review]
I feel this needs to be simplified and clarified. The last sentence feels very tautological.

File: source/node/writes.txt (hunk 1 above)
Comment (mongomoe) [reply source/node/writes.txt:256]: That does sound better. Taking the suggestion. Also, adding it to the react native doc as well.

**Comment by mongomoe** [comment]
@terakilobyte "I feel this needs to be simplified and clarified. The last sentence feels very tautological"

Thanks for the comment. I updated the paragraph to suggested phrasing.

**Review by terakilobyte — APPROVED** [review]
lgtm

**Pull request merged**